tests/test_calculator.cc

```diff
@@ -633,10 +633,12 @@ namespace rascal {
 
         size_t center_count{0};
         for (auto center : manager) {
+          auto half_it = manager_half->get_iterator_at(center_count, 0);
+          auto half_center = *(half_it);
```

Comment:
This is confusing me. What are half_center and how it differs from center is not obvious. What is happening?

Reply:
I'll see if i can find it out - that was @felixmusil 's doing. I guess he is referring to a half list and comparing it to a full list:
The iteration happens over `manager` while he also gets clusters from `manager_half`. And both the results of both of them are compared if they are the same?
I'll leave a comment to specify.